src/client/pages/Project/Changelog/index.js

```diff
@@ -99,6 +108,7 @@ class Changelog extends Component {
     }
 
     const { params: { id }, data } = this.props
+
```

Comment:
We should distinguish sorting and filtering. Selector returns sorted by commits date DESC list, but filtering by branch it's another story. Please, just fix selector.